docs/podman-commit.1.md

```diff
@@ -0,0 +1,95 @@
+% podman(1) podman-commit - Tool to create an image from a container's changes
+% Urvashi Mohnani
+# podman-commit "1" "December 2017" "podman"
+
+## NAME
+podman-commit - Create an image from a container's changes
+
+## SYNOPSIS
+**podman commit**
+**TARBALL**
+[**--author**|**-a**]
+[**--change**|**-c**]
+[**--message**|**-m**]
+[**--help**|**-h**]
+
+## DESCRIPTION
+**podman commit** creates an image from a container's changes. The author of the
+image can be set using the **author** flag. Various image instructions can be
+configured with the **--change** flag and a commit message can be set using the
+**--message** flag. The container is paused by default when committing, however
```

Comment:
OK, maybe I've PTO brain today, but I'm a bit confused.  I think we're pausing the original container by default and starting up the new one that is created based on the original + the changes.   If that's the case, that's not at all clear to me based on the doc that I'm seeing.

Reply:
Well it is actually pausing the container, so it does not change any content and then saving the containers content to an image.  After the save is done, the original container continues to run.

Reply:
What @rhatdan said. I reworded this, so hopefully it is clearer now.